index.bs

```diff
@@ -83,10 +83,10 @@ through a custom server.</p>
 
 <div class=note>
  This section replaces part of the WebSocket protocol opening handshake client requirement to
- integrate it with algorithms defined in Fetch. This way CSP, cookies, HSTS, and other Fetch-related
- protocols are handled in a single location. Ideally the RFC would be updated with this language,
- but it is never that easy. The WebSocket API, defined in the HTML Standard, has been updated to use
- this language. [[!WSP]] [[!HTML]]
+ integrate it with algorithms defined in the Fetch Standard. This way CSP, cookies, HSTS, and other
```

Comment:
Our usual style is `<cite>Fetch</cite>`, not `the Fetch Standard`.

Reply:
Should I also wrap the other uses of capitalized "Fetch" in a `<cite>`?

Reply:
Yeah, that'd make sense to me.